doc/sphinx/conf.py.in

```diff
@@ -41,6 +41,7 @@ rst_prolog="""
 .. |PCMK_INIT_ENV_FILE| replace:: ``%PACEMAKER_CONFIG_DIR%/pcmk-init.env``
 .. |PCMK_LOG_FILE| replace:: %CRM_LOG_DIR%/pacemaker.log
 .. |PCMK_CONTAINER_LOG_FILE| replace:: ``/var/log/pcmk-init.log``
+.. |PCMK__REMOTE_SCHEMA_DIR| replace:: %PCMK__REMOTE_SCHEMA_DIR%
```

Comment:
I wonder what happened, because I just did this a week ago.

Reply:
Either way thanks for fixing it :)

Reply:
Uhh https://github.com/ClusterLabs/pacemaker/commit/e4e0ed64ae6a2709883097a8059a49ed30869fcd